tests/integration/components/o-s-s/search-field-test.ts

```diff
@@ -0,0 +1,58 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, click, typeIn } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupIntl } from 'ember-intl/test-support';
+import Sinon from 'sinon';
```

Comment:
```suggestion
import sinon from 'sinon';
```
We don't use the uppercase for the `sinon` definition